public/javascripts/spec/e2e/editingTasksFeature.js

```diff
@@ -0,0 +1,27 @@
+describe('Editing a task', function() {
+
+  beforeEach(function() {
+    browser.get('http://localhost:8080');
+    element(by.model("toDoCtrl.newTask")).sendKeys('Make a to do list.');
+    element(by.buttonText("Add")).click();
+  });
+
+  it('allows a task to be edited via form when you click it', function() {
+    element(by.className('completed-false')).click();
+    element(by.model("task.text")).sendKeys(' And profit');
+    element(by.buttonText('Save')).click();
+    expect(element(by.className('completed-false')).getText()).toEqual('Make a to do list. And profit');
```

Comment:
Line is too long.<br>'expect' is not defined.<br>'element' is not defined.<br>'by' is not defined.